.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,3 +15,7 @@ Fixes #
 - [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
 - [ ] Write detailed docstrings for all functions/methods.
 - [ ] If adding new functionality, add an example to docstrings or tutorials.
+
+**Notes**
+
+- You can write `/foramt` in the first line of a comment to automatically format your codes
```

Comment:
```suggestion
- You can write `/format` in the first line of a comment to lint the code automatically
```

Fix a typo, and maybe use 'lint' instead of 'format'. Also, it's ['code', not 'codes'](https://english.stackexchange.com/questions/20455/is-it-wrong-to-use-the-word-codes-in-a-programming-context/) :wink: